projects/kit/tokens/input-files-content.ts

```diff
@@ -0,0 +1,26 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
```

Comment:
`link` and `label` are deprecated now, I don't think it makes sense to add token for them. Better to add `content` to `TuiInputFilesOptions` and make a default component as `content`:
```html
<a tuiLink>
    <ng-container *polymorpheusOutlet="computedLink$ | async as text">
        {{ text }}
    </ng-container>
</a>
<ng-container *polymorpheusOutlet="computedLabel$ | async as text">
    {{ text }}
</ng-container>
```
It can inject `InputFiles` and get `link` and `label` from it (use `Default` change detection strategy so it picks up the changes). We can also include `context` for that `content`, because the following properties might affect what we want to display:
- `multiple`
- `accept`
- `size`
- `maxFileSize`
- `fileDragged`